controllers/mapController.js

```diff
@@ -21,4 +23,29 @@ const getMapPoints = async (req, res, next) => {
   }
 };
 
+const createNewMap = async (req, res, next) => {
+  const { map, user } = req.body;
+
+  try {
+    const currentUser = await User.findOne({ _id: user }).exec();
```

Comment:
바디로 받는 User가 오브젝트 id 이므로

```suggestion
    const currentUser = await User.findById(user).exec();
```

이렇게 하는건 어떻게 생각하세요-?

바디로 받아온 user가 오브젝트 id라는걸 저희는 알지만 user라는 변수명으로는 직관적으로 알 수 없어서 혹시 나중에 좀 불편하려나요..?

Reply:
저도 objectId 를 사용해서 DB를 찾을때는 findById를 사용하는 쪽으로 통일하면 좋을 것 같다고 생각하는데요! 

위에서 `const { map, user: id } = req.body;` 로 받아서 `..... await Useer.findById(id).exec();` 이렇게 실행시켜주면 관련 메소드를 쓰면서도 직관적으로 수정할 수 있을 것 같은데 어떠실까요??